src/ontology/mondo-edit.obo

```diff
@@ -72135,17 +72135,30 @@ is_a: MONDO:0006215 {source="DOID:7222"} ! gallbladder adenocarcinoma
 id: MONDO:0004150
 name: breast giant fibroadenoma
 def: "A breast fibroadenoma characterized by a very large size. This term has also been used as a synonym for juvenile fibroadenoma by some authors. The latter is characterized by epithelial hyperplasia and an increased stromal cellularity." [NCIT:C4273]
+subset: gard_rare {source="GARD:20206"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:180267"}
+subset: orphanet_rare {source="Orphanet:180267"}
+subset: rare
```

Comment:
Should all these "subset" tags be added (similar to question on another PR)?

Reply:
@sabrinatoro should subset tags be moved over from a merged term or deleted?